### ISSUE TITLE
Add in Entropy

### ISSUE THREAD

**Issue by kennethwsmith** [Redmega/bitburner-save-editor#9]
A new multiplier was added with 1.6.0 Grafting feature called Entropy.  From the docs:

```
Be warned, some who have tested grafting have reported an unidentified malware. Dubbed “Entropy”, this virus seems
to grow in potency as more Augmentations are grafted, causing unpredictable affects to the victim.
```

It seems to degrade all other multipliers by 2% compounded.

I propose to add this to the PlayerSave tab.

**Comment by kennethwsmith** [comment]
See PR #10

**Comment by Redmega** [comment]
Closed by #10
